### ISSUE TITLE
Looking for 1.10.2 - Stuck at 1.7.10 due to lack of security.

### ISSUE THREAD

**Issue by P3rf3ctXZer0** [ForgeEssentials/ForgeEssentials#2166]
Thanks team.

**Comment by luacs1998** [comment]
1.10.2 port is in progress.

And what do you mean by "lack of security"?

**Comment by DPxKnightxZero** [comment]
Um not many mods like FE.

**Comment by olee** [comment]
Ok I want to ask as well - what do you mean with "lack of security" / "not many mods like FE".

If with the second one you mean mod compatibility, this should be pretty much fixed since a few weeks ago (found a general modcompat problem in FE that I fixed).

**Comment by Zethalion** [comment]
I think he means that there are not many mods like FE for server administration. On a 1.10 server there aren't many ways to protect things on a server. He would like to have FE released on 1.10 for this reason.

**Comment by DPxKnightxZero** [comment]
Exactly at least someone got my basic logic.

**Comment by MCTyler** [comment]
Many? Try None...Waiting on 1.10.2 also!

**Comment by spacebuilder2020** [comment]
Well, there is sponge.

**Comment by Psycholiquid71** [comment]
Hate to add to an old issue, but didnt want to repeat a new issue. Has there been any headway on the 1.10.2 port?

**Comment by spacebuilder2020** [comment]
http://ci.forgeessentials.com/job/FE-1.10/
